### ISSUE TITLE
CRITICAL: invalid cast from 'GtkEntry' to 'GtkTextView'

### ISSUE THREAD

**Issue by safocl** [lakinduakash/linux-wifi-hotspot#389]
**Describe the bug**
Application run returns 
```
(wihotspot-gui:58435): GLib-GObject-CRITICAL **: 15:03:23.958: invalid cast from 'GtkEntry' to 'GtkTextView'

(wihotspot-gui:58435): Gtk-CRITICAL **: 15:03:23.958: gtk_text_view_get_buffer: assertion 'GTK_IS_TEXT_VIEW (text_view)' failed

(wihotspot-gui:58435): GLib-GObject-CRITICAL **: 15:03:23.959: invalid (NULL) pointer instance

(wihotspot-gui:58435): GLib-GObject-CRITICAL **: 15:03:23.959: g_signal_connect_data: assertion 'G_TYPE_CHECK_INSTANCE (instance)' failed
/usr/bin/wihotspot: line 4: 58435 Segmentation fault      (core dumped) /usr/bin/wihotspot-gui
```

**To Reproduce**
1. Run application
2. See error

**Expected behavior**
Launching the application

**Screenshots**
[Screenshot](https://i.imgur.com/GvBUxhl.png)

**Desktop (please complete the following information):**
 - OS: archlinux
 - main
 - gtk3 1:3.24.41-1

**Additional context**
this error occurs because in the ui.c file GtkEntry is cast to GtkTextView:
https://github.com/lakinduakash/linux-wifi-hotspot/blob/fe64e7113a8564c78b67f7b32d5e606e1b631c65/src/ui/ui.c#L426
but https://github.com/lakinduakash/linux-wifi-hotspot/blob/fe64e7113a8564c78b67f7b32d5e606e1b631c65/src/ui/glade/wifih.ui#L377 (entry_gateway is a GtkEntry, not GtkTextView)

**Comment by safocl** [comment]
[TextBuffer](https://docs.gtk.org/gtk3/class.TextBuffer.html) and [EntryBuffer](https://docs.gtk.org/gtk3/class.EntryBuffer.html) are not compatible.

**Comment by safocl** [comment]
create PR: https://github.com/lakinduakash/linux-wifi-hotspot/pull/390

**Comment by lakinduakash** [comment]
Merged PR #391 ,
Thanks
